src/next/scval_conversions.rs

```diff
@@ -303,15 +303,15 @@ impl TryFrom<ScVal> for ScSymbol {
 #[cfg(feature = "alloc")]
 impl TryFrom<String> for ScVal {
     type Error = ();
-    fn try_from(v: String) -> Result<Self, Self::Error> {
+    fn try_from(v: String) -> Result<Self, ()> {
```

Comment:
What's the significance of this change? I don't think there is any significance, just wondering.

Reply:
They're all just disambiguation because Self::Error is ambiguous.